src/DetailsView/reports/components/outcome-type.ts

```diff
@@ -4,16 +4,16 @@ import { countBy, values } from 'lodash';
 
 import { ManualTestStatus, ManualTestStatusData } from '../../../common/types/manual-test-status';
 
-export type OutcomeType = 'pass' | 'incomplete' | 'fail';
+export type RequirementOutcomeType = 'pass' | 'incomplete' | 'fail';
```

Comment:
It seems strange to have one kind of outcome type here (`RequirementOutcomeType` in outcome-type.ts), the other kind in a component (`InstanceOutcomeType` in outcome-summary-bar.tsx) and the joint one in a third component (`OutcomeType` in outcome-chip.tsx). I would be tempted to either have the three OutcomeTypes in one place (outcome-type.ts) or have separate files for each kind of OutcomeType (outcome-type.ts, requirement-outcome-type.ts, instance-outcome-type.ts).
If we keep this file just having `RequirementOutcomeType` then it should rename to requirement-outcome-type.ts.